request/request.d.ts

```diff
@@ -86,6 +86,7 @@ declare module 'request' {
 			qs?: any;
 			json?: any;
 			multipart?: RequestPart[] | Multipart;
+            agent?: http.Agent;
```

Comment:
The documentation for this option is as follows:

> `agent` - `http(s).Agent` instance to use

thus shouldn't the type annotation be `http.Agent | https.Agent`?